starter-code/src/components/ProductRow.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+class ProductRow extends Component {
+  render() {
+    return (
+      <tr>
+        <td
+          style={
+            this.props.myProduct.stocked === true
```

Comment:
Il n'est pas nécessaire de faire `=== true` car on sait que stocked est une variable booléenne

Etant donné que la couleur par défaut du texte est `black`, tu aurais pu remplacer cette expression ternaire par:
`style={this.props.myProduct.stocked || { color: "black" }}` ou bien par `style={!this.props.myProduct.stocked && { color: "black" }}`